mod/item.php

```diff
@@ -359,21 +367,20 @@ function item_post(App $a) {
 	$self   = false;
 	$contact_id = 0;
 
-	if((local_user()) && (local_user() == $profile_uid)) {
+	if ((local_user()) && (local_user() == $profile_uid)) {
 		$self = true;
 		$r = q("SELECT * FROM `contact` WHERE `uid` = %d AND `self` = 1 LIMIT 1",
 			intval($_SESSION['uid']));
-	}
-	elseif(remote_user()) {
-		if(is_array($_SESSION['remote'])) {
-			foreach($_SESSION['remote'] as $v) {
-				if($v['uid'] == $profile_uid) {
+	} elseif(remote_user()) {
```

Comment:
Maybe like this: `if (x($_SESSION, 'remote') && is_array($_SESSION['remote'])) {`